bin/configs/ktorm-schema.yaml

```diff
@@ -0,0 +1,6 @@
+generatorName: ktorm
```

Comment:
For the generator name, please use `ktorm-schema` to make it consistent with other schema generators (e.g. graphql, protobuf, etc)